Jobsway2go.Core/Interfaces/ICollectionService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Jobsway2go.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Jobsway2go.Core.Interfaces
+{
+    public interface ICollectionService : IRepository<Collection>
+    {
+        public void SaveToCollection();
+    }
```

Comment:
1. Remove `public` keyword
2. Add `Remove` method (or `Delete`, however you want to call it)